vsxexport.sh

```diff
@@ -476,7 +485,12 @@ else
     printf "| \t\t\t| ipassignment.conf NOT found\t\t|${txt_green} OK${txt_reset}\t\t|\n"
 fi
 
-
+if [[ -e $FWDIR/conf/rad_conf.C ]]; then
+    cp --parents $FWDIR/conf/rad_conf.C $OUTPUTDIR/VS0
+    printf "| \t\t\t| rad_conf.C found\t\t\t|${txt_green} SAVED${txt_reset}\t\t|\n"
+else
+    printf "| \t\t\t| rad_conf.C NOT found\t\t|${txt_green} OK${txt_reset}\t\t|\n"
```

Comment:
Extra \t was needed here. Needs to be changed into:
printf "| \t\t\t| rad_conf.C NOT found\t\t\t|${txt_green} OK${txt_reset}\t\t|\n"